Tutorials/Creating-a-Custom-Dashboard/index.md

```diff
@@ -3,11 +3,12 @@ product: "CMS"
 meta.Title: "Creating a Custom Dashboard"
 meta.Description: "A guide to creating a custom dashboard in Umbraco"
 versionFrom: 8.0.0
+verified-against: 9.0.0
 ---
 
 # Tutorial - Creating a Custom Dashboard
 
-_This tutorial was last tested on **Umbraco 8.11**_
+_This tutorial was last tested on **Umbraco 9.0.0**_
```

Comment:
As Umbraco 9 is not released yet, I think it makes sense if we continue to mention which version of v8 this tutorial was last tested on.

```suggestion
_This tutorial was last tested on **Umbraco 8.11** and **Umbraco 9.0.0**_
```